target/linux/ramips/dts/ZBT-WG3526.dtsi

```diff
@@ -0,0 +1,118 @@
+#include "mt7621.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+/ {
+	compatible = "mediatek,mt7621-eval-board", "mediatek,mt7621-soc";
```

Comment:
Then drop it. I guess it was meant as ```mediatek,<actual_board>``` and was just copied from the eval board dts to the next new board and so on.